weblate/checks/chars.py

```diff
@@ -248,6 +249,11 @@ def _check_my(self, source, target):
     def check_single(self, source, target, unit):
         if not source or not target:
             return False
+        if (
+            source[-2:] in {"?!", "!?", "？！", "！？"}
+            or target[-2:] in {"?!", "!?", "？！", "！？"}
+        ) or (source[-1] in {"⁈", "⁉"} or target[-1] in {"⁈", "⁉"}):
```

Comment:
This can be much simpler:

```suggestion
        if source.endswith(interrobangs) or target.endswith(interrobangs):
```